src/test/java/bio/terra/pact/provider/DataRepoProviderTest.java

```diff
@@ -57,6 +59,13 @@ class DataRepoProviderTest {
   @MockBean private AuthenticatedUserRequestFactory authenticatedUserRequestFactory;
   @MockBean private AssetModelValidator assetModelValidator;
 
+  @PactBrokerConsumerVersionSelectors
+  public static SelectorBuilder consumerVersionSelectors() {
+    // Select consumer pacts published from default branch or pacts marked as deployed or released
+    // aj-1188-tdr-consumer is included here for development purpose + PR runs
+    return new SelectorBuilder().mainBranch().deployedOrReleased().branch("aj-1188-tdr-consumer");
+  }
+
```

Comment:
@ichengchang I'm curious as to what the best practice normally is here:

If we're not trying to test out a specific consumer feature branch in flight, do we need this method at all?  Or would we want to check in something like this for standard Pact verification runs?
```
  @PactBrokerConsumerVersionSelectors
  public static SelectorBuilder consumerVersionSelectors() {
    // Select consumer pacts published from default branch or pacts marked as deployed or released
    return new SelectorBuilder().mainBranch().deployedOrReleased();
  }
```

Reply:
The default branch of the provider repository should be using `SelectorBuilder().mainBranch().deployedOrReleased()` to ensure that it verifies pacts generated by the head of the default consumer branch (`mainBranch()`) or pacts marked with one of these `env` tags: `env: alpha`, `env: dev`, `env: staging`, or `env: prod` (`deployedOrReleased()`).

During development, you can temporarily add a specific branch to the selector and use it to test provider code against the pact identified by the selected branch through PRs. When everything looks good and the consumer branch has been merged. the selector is restored to `SelectorBuilder().mainBranch().deployedOrReleased()` before merging the provider code.